src/helpers.ts

```diff
@@ -93,6 +93,11 @@ export function normaliseOptions(
   return options
 }
 
+function isHighSurrogate(char: string): boolean {
+  const charCode = char.charCodeAt(0)
+  return charCode >= 0xd800 && charCode <= 0xdbff
```

Comment:
Just thinking that I can possibly change that into something such as:

```suggestion
  return char >= '\ud800' && char <= '\udbff'
```

Shorter, avoiding a call to charCodeAt... WDYT @keithamus?

Reply:
:+1:

Reply:
I'll benchmark the two functions before changing it just in case

Reply:
i did a quick benchmark of the two and it seems the string comparison is faster. interesting :D 

i suppose internally it still gets the char code but in c++ or whatever, good shout

Reply:
So I'll go with the string comparison then. I still have to re-lint before my PR passes but let's update that part